### ISSUE TITLE
Can I change the camera's parameters on ROS Noetic for the autorace 2020?

### ISSUE THREAD

**Issue by fabiocyro** [ROBOTIS-GIT/emanual#1463]
Hello,

I had a turtlebot3 with me operating with ROS Kinetic and I was using the autorace package to learn, and the step of changing the camera's parameters was crucial for me to achieve the lane detection part. I'm working on a controlled light environment and I had the optimal configuration to achieve everything.

Now I am using ROS Noetic and I can't seem to be able to do that anymore.

To better illustrate:
![crop2](https://user-images.githubusercontent.com/37183299/156018255-6679fcba-1521-4e30-a9f8-140aaa611537.png)

In other words, I would like also to change these parameters (iso, saturation etc)

And this is what I get on Noetic:

![crop1](https://user-images.githubusercontent.com/37183299/156018586-cde8fc4b-7c7f-4cd5-b6c1-5dd6ffc8a59a.png)

Is there any solution?

Thanks in advance!

**Comment by ROBOTIS-Yang** [comment]
Hi, fabiocyro

Given the extended period of inactivity, we're marking this issue as closed. If you believe this issue is still relevant, please let us know and we'll reopen it.
